.gitignore

```diff
@@ -10,14 +10,5 @@
 *.swp
 *.DotSettings.user
 /packages
-/Test/Capture_img
-/Test/Config*.ini
-/Test/bs1770gain**
-/Test/dll/CSharpTest.Net.Collections.*
-/Test/dll/FDK.*
-/Test/dll/Newtonsoft.Json.*
-/Test/dll/Sentry.*
-/Test/dll/SlimDX.*
-/Test/dll/System.*
-/Test/TJAPlayer3.exe.config
+/Test
```

Comment:
I'd like to retain the previous .gitignore for now as there are sadly still some files within the Test folder that are under source control and need to be monitored. Work on #10, especially if supported by library updates, should help open things up for improvement in that area.